bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/DPIZoomChangeHandler.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2024 Yatta Solutions and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Yatta Solutions - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.internal;
+
+import org.eclipse.swt.widgets.*;
+
+@FunctionalInterface
+public interface DPIZoomChangeHandler {
+	public void handleDPIChange(Widget widget, int newZoomFactor, float scalingFactor);
```

Comment:
Reason is you never need the oldZoomFactor anywhere, but the scalingFactor. So, as a result, I would need to add the scalingFactor calculation in multiple places. But, I understand the reasoning of having two parameters with the same unit instead of two different units. I would not argue strongly against it.

Reply:
I understand, then I would expect
1. a documentation of that method with an explanation of its parameters,
2. a proper naming to distinguish the two kinds of "factors". Even though I know we have discussed something different before: DPIUtil always simply uses "zoom" for zoom factors, so maybe that's the common term to distinguish from actual "factors"?